public/app/percona/dbaas/DBaaS.messages.tsx

```diff
@@ -53,6 +54,23 @@ export const Messages = {
       [KubernetesClusterStatus.unavailable]: 'Unavailable',
       [KubernetesClusterStatus.invalid]: 'Invalid',
     },
+    updateOperatorModal: {
+      cancel: 'Cancel',
+      confirm: 'Update',
+      title: 'Confirm operator update',
+      buildUpdateOperatorMessage: (
+        operatorType: string,
+        newVersion: ReactNode,
+        kubernetesClusterName: ReactNode,
+        currentVersion?: string
+      ) => (
+        <>
+          Are you sure you want to update {operatorType} {currentVersion} to version {newVersion} in{' '}
```

Comment:
Could we send this as a function to the messages file?

Reply:
I'm not understanding the question @fabio-silva sorry. This is a function and this is the messages file 🤔

Reply:
You're totally right. Sorry, let's pretend I never said this 😂